server/scripts/poracleToGeoJSON.js

```diff
@@ -0,0 +1,68 @@
+/**
+* Credits: https://gist.github.com/moriakaice
+* Editor: PJ0tterr
+* Date: 15-01-2022
+*/
+
+const fs = require('fs');
+const path = require('path');
+
+// Set Path where for area.json
+const configFolder = path.resolve(__dirname, '../../src/configs')
+const geofencesFolder = path.resolve(__dirname, '../../public/geofence/geofence.json')
+
+if (!fs.existsSync(geofencesFolder)) {
+  // eslint-disable-next-line no-console
+  console.error('Error: Geofence directory does not exist:', geofencesFolder);
+  return;
+}
+
+const outGeoJSON = {
+  type: 'FeatureCollection',
+  features: [],
+};
+
+fs.readFile(geofencesFolder, 'utf8', (err, data) => {
+  if (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
+    return;
+  }
+  const inGeoJSON = JSON.parse(data);
+  if (inGeoJSON.length === 0) {
+    // eslint-disable-next-line no-console
+    console.error('Failed to parse poracle geofence file');
+    return;
+  }
+  // eslint-disable-next-line no-plusplus
```

Comment:
For ESlint console warnings, you can disable them for the whole file instead of line by line, but please for the rest of them, fix the code instead of skipping the rule. Please also remove `;` as they aren't used in this codebase.

Reply:
Read the information about it and add ESLint to code and remove the ; from the code.